POMs/offerModalPage.ts

```diff
@@ -0,0 +1,90 @@
+import { expect, Locator, Page } from '@playwright/test';
+
+export class OfferModalPage {
+
+    readonly page: Page;
+    readonly newOfferModal: Locator;
+    readonly createOfferButtton: Locator;
+    readonly clientDropdown: Locator;
+    readonly projectDropdown: Locator;
+    readonly languageDropdown: Locator;
+    readonly cancelButton: Locator;
+    readonly continueButton: Locator;
+    readonly clientZeroProjectsError: Locator;
+    readonly pageBar: Locator;
+    readonly cobeLogo: Locator;
+
+
+
+
+    constructor(page: Page) {
+        this.page = page;
+        this.newOfferModal = page.getByText('Create New Offer');
+        this.createOfferButtton = page.getByRole('button', { name: 'Create new offer' });
+        this.clientDropdown = page.locator('.css-9uxpgx');
+        this.projectDropdown = page.locator('#react-select-2-placeholder');
+        this.languageDropdown = page.locator('#react-select-3-placeholder');
+        this.cancelButton = page.getByRole('button', { name: 'Cancel' });
+        this.continueButton = page.getByRole('button', { name: 'Continue' });
+        this.clientZeroProjectsError = page.locator('div').filter({ hasText: /^Selected client is without a project, please create a project first\.$/ }).first();
+        this.pageBar = page.getByText('ProjectProject');
+        this.cobeLogo = page.locator('svg').first();
+
+
+    }
+
+    async selectClient(client: string) {
+        await this.createOfferButtton.click();
+        await this.page.waitForTimeout(300);
+        await this.clientDropdown.click();
+        await this.page.getByText(client, { exact: true }).click();
+    }
+
+    async assertClientZeroProjectsError() {
+        await expect(this.clientZeroProjectsError).toBeVisible();
+    }
+
+    async selectProject(project: string) {
+        await this.page.waitForTimeout(300);
+        await this.projectDropdown.click();
+        await this.page.getByText(project, { exact: true }).click();
+
+    }
+
+
+    async selectLanguage(language: string) {
+        await this.languageDropdown.click();
+        await this.page.waitForTimeout(300);
+        await this.page.getByText(language, { exact: true }).click();
+
+    }
+
+    async assertDisabledContinueButton() {
+        await this.page.locator('.sc-ftvSup > div:nth-child(2)').isDisabled();
+    }
+
+    async assertAutomaticProjectFill(project: string) {
+        await this.page.waitForTimeout(300);
+        expect(this.page.getByText(project, { exact: true }));
+
+    }
```

Comment:
This solution works, but it can also be checked if the projectBar is hidden
When the project is preset, project bar is not clickable, and then you don't need to pass the project as the function argument